tests/test_apiclient.py

```diff
@@ -2217,11 +2217,14 @@ def test_find_services_iter_additional_arguments(self, data_client, rmock):
 
         assert len(results) == 2
 
+    @pytest.mark.parametrize("status_code", [
+        (502), (503), (504)
```

Comment:
Parentheses around numbers don't do anything in this case, it's safe to remove them